samples/client/petstore/R-httr2-wrapper/.openapi-generator/FILES

```diff
@@ -5,6 +5,7 @@
 .travis.yml
 DESCRIPTION
 NAMESPACE
+README.md
```

Comment:
seems like the sort ordering is not kept stable across systems for this one

Reply:
yes, probably because the samples were generated on a windows machine

Reply:
can you fix it?

Reply:
trying already. Always tricky to do non-platform specific things :)

Reply:
https://github.com/OpenAPITools/openapi-generator/pull/19726